src/bundle/DependencyInjection/Configuration/Parser/ContentTypeGroup.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\AdminUi\DependencyInjection\Configuration\Parser;
+
+use Ibexa\Bundle\Core\DependencyInjection\Configuration\AbstractParser;
+use Ibexa\Bundle\Core\DependencyInjection\Configuration\SiteAccessAware\ContextualizerInterface;
+use Symfony\Component\Config\Definition\Builder\NodeBuilder;
+
+/**
+ * Configuration parser for content type group icons.
+ *
+ * Example configuration:
+ *
+ * ```yaml
+ * ibexa:
+ *   system:
+ *      default: # configuration per siteaccess or siteaccess group
+ *          content_type_group:
```

Comment:
Note: The more those appear, the more I think that we should not pack everything under scope root. It's quite likely that core-related package might want to introduce the same key in the future. It rather should be
```yaml
ibexa:
    system:
        default:
            admin_ui: # generic parser with AdminUI extension point
                content_type_group:
```
to avoid the situation we recently had with `notifications` duplicate key.

Reply:
sure, but we already have a parser for content type icons with 
```
   system:
      default: # configuration per siteaccess or siteaccess group
          content_type:
             article:
                thumbnail: '/assets/images/customarticle.svg'
             poll:
                thumbnail: '/assets/images/poll.svg'
```